content/finalist/ttjc@finalist@swapnadeepmohapatra/index.md

```diff
@@ -6,10 +6,11 @@ portfolio: "https://www.swapnadeep.com"
 github-link: "https://www.github.com/swapnadeepmohapatra"
 linkedin-link: "https://www.linkedin.com/in/swapnadeep-mohapatra-11b371159/"
 looking-for: "internship"
+image-link: https://avatars2.githubusercontent.com/u/41564532?s=460&u=ac84aa8f047ebb85f5ba8f9e77eca42d961dcf5c&v=4
```

Comment:
Add link inside double quotes "  "

Reply:
@gkanishk I have fixed it please have a look.